packages/image/src/ambient.d.ts

```diff
@@ -0,0 +1,12 @@
+declare module '__svelte-image-options__.js' {
```

Comment:
@dummdidumm I get all kinds of errors in `Image.svelte` in VS Code because this isn't being picked up. Is it working for you? Any idea what I might need to do? It works if I put in in the `types` folder.